src/main/resources/application.properties

```diff
@@ -1,11 +1,12 @@
-sprint.datasource.url=jdbc:h2:mem:mydb
-sprint.datasource.driver-class-name=org.h2.Driver
-sprint.datasource.username=root
-sprint.datasource.password=root
+spring.datasource.url=jdbc:postgresql://localhost:5432/mydb
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.username=postgres
+spring.datasource.password=root
```

Comment:
It's not a good practice to store credentials in the repo. It would be nice to store them in environment variables.

Reply:
The good point, but it's the test project. I think, we can keep it without any changes.

Of course in production we must use other ways to get secret values for the db connection like AWS SecretsManager or ENV variables as mentioned above.